libbeat/docs/shared-template-load.asciidoc

```diff
@@ -82,3 +116,12 @@ at the newest documents. Use this command:
 ----------------------------------------------------------------------
 curl -XDELETE 'http://localhost:9200/{beatname_lc}-*'
 ----------------------------------------------------------------------
+
+[[passing-credentials-template-loading]]
+==== Pass credentials
+
+//REVIEWERS: How do you do this when you are using Winlogbeat and starting as a service? Do you need to configure username/password in the config file? I haven't mentioned the config options in the GS because I think it's better for users not to store the passwords in config files. Is that a correct assumption?
```

Comment:
@monicasarbu I've added a section to the configuration steps that explains how to specify username and password in the config file and kept the examples for setting the options from the command line. We could simplify the whole thing by not mentioning how to set the options from the command line, but I think some people might want to do it that way. WDYT?